### ISSUE TITLE
Multiple text properties/dynamic text input?

### ISSUE THREAD

**Issue by sorenbug** [andlabs/libui#466]
Following up from #174, as I can't find the `utflib-and-attrstr` branch, what if we were able to create some sort of input where certain parts of text were editable, styled, etc and others weren't in one sort of "text buffer"? (A.k.a. how to create an emacs-style buffer system with libui)

What would might this code look like in the libui style? What would it take to implement?

**Comment by andlabs** [comment]
Yes, I originally started work with attributed strings specifically as the precursor to a text input mechanism for uiArea. That will have to wait for after Alpha 5 to be finished, so stay tuned.
